lib/acl.c

```diff
@@ -43,17 +43,38 @@ enum {
 	NUM_INPUTS_IPV6,
 };
 
-/* Callback function for when there's no classification match. */
+/* TODO Parse the unmathced packets, and direct them if necessary. */
```

Comment:
In fact, the callback function ``ext_cb_func`` does the job here. So, I removed the comment.